jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/panel/DynamicAttributesPanel.java

```diff
@@ -16,275 +16,38 @@
 
 package io.jmix.dynattrflowui.panel;
 
-import com.google.common.base.Strings;
 import com.vaadin.flow.component.Component;
-import com.vaadin.flow.component.Composite;
+import com.vaadin.flow.component.HasComponents;
 import com.vaadin.flow.component.HasSize;
-import com.vaadin.flow.component.Text;
-import com.vaadin.flow.component.formlayout.FormLayout;
-import com.vaadin.flow.component.html.H3;
+import com.vaadin.flow.component.combobox.ComboBox;
 import com.vaadin.flow.component.orderedlayout.VerticalLayout;
-import io.jmix.core.DevelopmentException;
-import io.jmix.core.Messages;
-import io.jmix.core.metamodel.model.MetaProperty;
-import io.jmix.dynattr.AttributeDefinition;
-import io.jmix.dynattr.CategoryDefinition;
-import io.jmix.dynattr.DynAttrMetadata;
-import io.jmix.dynattr.DynAttrQueryHints;
-import io.jmix.dynattr.model.Categorized;
 import io.jmix.dynattr.model.Category;
-import io.jmix.flowui.UiComponents;
-import io.jmix.flowui.component.ComponentGenerationContext;
-import io.jmix.flowui.component.UiComponentsGenerator;
-import io.jmix.flowui.component.combobox.JmixComboBox;
 import io.jmix.flowui.component.formlayout.JmixFormLayout;
-import io.jmix.flowui.data.ValueSource;
-import io.jmix.flowui.data.value.ContainerValueSource;
-import io.jmix.flowui.data.value.ContainerValueSourceProvider;
-import io.jmix.flowui.model.DataLoader;
-import io.jmix.flowui.model.HasLoader;
 import io.jmix.flowui.model.InstanceContainer;
-import io.jmix.flowui.view.ViewValidation;
 
 import javax.annotation.Nullable;
-import java.util.*;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
+import java.util.List;
 
-public class DynamicAttributesPanel extends Composite<VerticalLayout> implements HasSize {
+public interface DynamicAttributesPanel extends HasSize, HasComponents {
```

Comment:
Most of Jmix components don't have an interface. I don't see reasons to have it for dyn attrs panel